### PR TITLE
Only process XML documents with the schemes 'file' and 'untitled'

### LINKED ISSUE #861: DTD Validation and GIT Repository throws errors in GIT files
Hi,

easy to reproduce:

Create a new folder:

Create a DTD file person.dtd

`<!ELEMENT person ( first_name) >`

Create a XML file person.xml with a reference to a DTD file

`<?xml version="1.0" ?>`
`<?xml-model href="person.dtd" ?>`
`<person>`
            `    <first_name>Michael</first_name>`
`</person>`

Go to terminal and type

git init
git add * 
git commit -m"test"

Now vscode will show you a problem in file

person.xml.git
- Cannot find DTD 'git:/c%3a/Users/path_to_my_folder/person.dtd

which normally does not exists

### PR DESCRIPTION
**Pull request by datho7561**
Fixes #861

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

**Review by JessicaJHee — APPROVED** [review]
LGTM! Fixes the issue and works as expected.

File: tsconfig.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "alwaysStrict": true,
     "lib": [
       "es6",
-      "dom"
+      "dom",
+      "es2019"
```
Comment (fbricon) [inline tsconfig.json:10]: why is this necessary? Either use es2019 or es6, not both
